### PR TITLE
LPS-120493 Pasting screenshot in Alloy Editor is not working

### PR DESCRIPTION
**Pull request by markocikos**
This is a regression bug fix, see details and steps to reproduce in https://issues.liferay.com/browse/LPS-120493. Note - the steps in descriptions are not correct, correct ones are in [this comment](https://issues.liferay.com/browse/LPS-120493?focusedCommentId=2242810&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2242810): the bug is that screenshot paste is not working, not copy & paste from file system.

After fix:
screenshot

As in gif, testing for this PR should include two other closely related use cases:
1. Drag & drop of image from file system into editor.
2. Copy & paste of image from a web page or Excel into editor. 

Notes:
- The root cause of the bug is that events need to be defined on `editor.editable` object, and not on `editor`. We changed this in migration to simplify code, but it was a mistake. There is an internal CKEditor mechanism, in `clipboard` plugin, that pre-processes paste data and sets event facade, and ordering events execution expects events on `editable`. In this PR, we are reverting handle definitions to be the same as in the original plugin.
- I snuck in some optional chaining :muscle: 
- I needed to change trigger condition for copy & paste html case. @julien please double check this one.

### CODE REVIEW

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/addimages/plugin.js [hunk 1]
```diff
@@ -192,23 +192,20 @@
...
-				event.data.$.clipboardData.items.length > 0
-			) {
-				const pastedData = event.data.$.clipboardData.items[0];
-
-				if (pastedData.type.indexOf('image') === 0) {
-					const imageFile = pastedData.getAsFile();
-
-					this._processFile(imageFile, event.editor);
-				}
+			if (!event.data?.$?.clipboardData?.items) {
```
Comment (julien) [inline modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/addimages/plugin.js:195]: @markocikos  nice

**Comment by markocikos** [comment]
ci:test:sf

**Comment by markocikos** [comment]
ci:test:relevant

**Comment by julien** [comment]
@markocikos lg2m

**Comment by julien** [comment]
ci:forward

**Pull request closed without merging**